nationalarchives/tdr-transfer-frontend
# Show partially complete error

- Make consignment reference not optional
- Bump to latest version of generated graphql
- Add upload in progress page and messages
- Add new endpoint and route
- Add consigment status service and tests
- Add in consignment status check to action
- Add new tests for the upload endpoint
